README.md

```diff
@@ -1,17 +1,22 @@
+# Moved to [`github.com/containerd/containerd/pkg/cri`](https://github.com/containerd/containerd/tree/master/pkg/cri)
+
+On October 7, 2020, the contents of this repo were moved to [`pkg/cri` package](https://github.com/containerd/containerd/tree/master/pkg/cri)
+in [the main containerd repo](https://github.com/containerd/containerd).
+
+**Pull requests are no longer accepted in the master branch of this repo.**
+PRs for `release/1.3` and `release/1.4` branches are still accepted, but the commits must be merged into the
+main repo first, and then backported to this repo.
```

Comment:
s/accepted,/, e.g. for service updates./
s/but the commits must be merged into the main repo/However, the master branch for containerd/cri integration work is now located in the containerd/containerd repository, and as such new commits should be merged there/

Reply:
What do you mean by "service updates"?

Reply:
I mean the 1.3 and 1.4 branches are service branches.. 
By service updates I mean updates to fix bugs, improve performance, and resolve security issues.

Reply:
Updated. I think we can simply call it "bug-fix PRs". PTAL.